share/lib/python/scripts/_binwrapper.py

```diff
@@ -7,7 +7,7 @@
 import shutil
 import subprocess
 import sys
-from pkg_resources import working_set
+from importlib.metadata import distributions
```

Comment:
`importlib.metadata` is only present in Python 3.9 and above; since I see NEURON nightly on PyPI still has a 3.8 version available for download, you can use the backported `importlib_metadata` third-party module for it.
Note that then we also need to update the NEURON recipe for Spack (I think it's [this file](https://github.com/BlueBrain/spack/blob/develop/bluebrain/repo-patches/packages/neuron/package.py)).

Reply:
Thanks I added the backport package.